app/tests/unit/components/Form/ProjectAnnualReportForm.test.tsx

```diff
@@ -252,7 +252,7 @@ describe("The ProjectAnnualReportForm", () => {
     });
   });
 
-  it("undoes all annual report changes (resets the form to empty) when the user clicks the Undo Changes button while creating a project", () => {
+  it("calls the undoFormChangesMutation when the user clicks the Undo Changes button", () => {
```

Comment:
I updated this test suite because I liked the way I figured out how to do it for the milestone report better. Checking undoing on creating vs. editing I'll address with a cypress test. Creating is in this PR; editing will be a separate one.